### ISSUE TITLE
indented arguments weirdness

### ISSUE THREAD

**Issue by ddlsmurf** [clutchski/coffeelint#4]
I'm not sure this is a bug, it smells weird to me, thought I'd report

```
obj
  .method
    param: "val" # Line contains inconsistent indentation
obj.
  method
    param: "val" # Line contains inconsistent indentation
obj
  .method(
    param: "val" # Line contains inconsistent indentation
  )

obj
.method
  param: "val" # Line contains inconsistent indentation

obj.
method
  param: "val" # Line contains inconsistent indentation

obj
.method(
  param: "val" # Line contains inconsistent indentation
)

obj
  .method()
obj.
  method()
obj
.method()
obj.
method()

a = () ->
  obj # Line contains inconsistent indentation
  .method
    param: "val"
  obj
  .method
    param: "val"
a = () ->
  obj. # Line contains inconsistent indentation
  method
    param: "val"
  obj.
  method
    param: "val"
a = () ->
  obj # Line contains inconsistent indentation
  .method(
    param: "val"
  )
  obj
  .method(
    param: "val"
  )
a = () ->
  obj # Line contains inconsistent indentation
  .method(
    param: "val"
  )
  obj
  .method(
    param: "val"
  )
```

**Comment by ibrahima** [comment]
Is this supposed to be flagged? It seems like whether or not the below code is flagged depends on whether the dots appear on even column numbers or odd (odd numbers trigger the error):

```coffeescript
$('#some_element').attr('disabled', false)
                  .removeClass('disabled')
                  .val(someValue)
```

We follow this style of indentation for our Ruby code as well (using Rubocop) and I couldn't figure out if there was a linter option for this or not, but it seems like this style of indentation is sometimes marked as valid and sometimes invalid depending on what column the first dot appears at.
